packages/api/src/command/medical/facility/create-facility.ts

```diff
@@ -32,19 +33,25 @@ export const createFacility = async ({
     data,
   };
   validateCreate(input);
+  const facilityByNpi = await getFacilityByNpi({ cxId, npi: data.npi });
+  if (facilityByNpi) {
+    throw new BadRequestError(
+      `Can't create a new facility with the same NPI as facility with ID: ${facilityByNpi.id} and name: ${facilityByNpi.data.name}`
+    );
+  }
   return FacilityModel.create(input);
-};
+}
 
 export function validateCreate(facility: FacilityCreate, throwOnError = true): boolean {
-  const { cwType, cqType, cqActive, cwActive, cqOboOid, cwOboOid } = facility;
-  if (isOboFacility(cwType) && cwActive && !cwOboOid) {
+  const { cwType, cqType, cqOboOid, cwOboOid } = facility;
+  if (isOboFacility(cwType) && !cwOboOid) {
```

Comment:
Assuming we removed active check to create facility that are not active

Reply:
Yes.